server/policy.go

```diff
@@ -148,6 +148,17 @@ func (s *Server) registerPolicyRoutes(g *echo.Group) {
 		policyFind := &api.PolicyFind{
 			Type: &pType,
 		}
+		if c.QueryParam("resourceType") != "" {
```

Comment:
How about use `getPolicyResource` here to format its value?

Reply:
c.QueryParam and c.Param are different things, what are we looking for exactly?

Reply:
Good suggestion. Updated, QueryParam is used for filtering.

Reply:
I am still confused,

Should it be c.Param("resourceType"), same for resourceId

Or put another way, what's the point of having query parameter resourceId and resourceType when those two are already in the path